tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/pages/trusts/ofstedPage.ts

```diff
@@ -119,47 +130,47 @@ class OfstedPage {
     public checkCurrentRatingsQualityOfEducationJudgementsPresent(): this {
         this.elements.currentRatings.qualityOfEducation().each((element) => {
             const text = element.text();
-            expect(text).to.match(/Good|Outstanding|No judgement|Outstanding|Requires improvement|Inadequate|Not yet inspected/);
+            expect(text).to.match(/Good|Outstanding|No judgement|Outstanding|Requires improvement|Inadequate|Not yet inspected|Insufficient evidence/);
```

Comment:
Outstanding is duplicated in the ofsted ratings. I would suggest moving this regex out to be part of the page object model class, use the dateRegex as an example on that.

Reply:
Now added this